test/Facilitator/confirmRedeemIntent.js

```diff
@@ -242,12 +262,35 @@ describe('Facilitator.confirmRedeemIntent()', () => {
         redeemParams.gasPrice,
         redeemParams.gasLimit,
         redeemParams.hashLock,
+        redeemParams.blockNumber,
         txOptions,
       ),
       `Invalid facilitator address: ${txOptions.from}.`,
     );
   });
 
+  it('should throw an exception when available state root on target is lower than the height at which redeem was done.', async () => {
```

Comment:
Add another test for the validation of block number.

Reply:
Done!